arwen/elrondapi/generateMain/generateApi.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"go/token"
+	"os"
+
+	eapigen "github.com/ElrondNetwork/wasm-vm/arwen/elrondapi/generate"
+)
+
+const pathToElrondApiPackage = "./"
+
+func initEIMetadata() *eapigen.EIMetadata {
+	return &eapigen.EIMetadata{
+		Groups: []*eapigen.EIGroup{
+			&eapigen.EIGroup{SourcePath: "bigFloatOps.go"},
+			&eapigen.EIGroup{SourcePath: "bigIntOps.go"},
+			&eapigen.EIGroup{SourcePath: "elrondei.go"},
+			&eapigen.EIGroup{SourcePath: "generateOps.go"},
+			&eapigen.EIGroup{SourcePath: "managedei.go"},
+			&eapigen.EIGroup{SourcePath: "manBufOps.go"},
+			&eapigen.EIGroup{SourcePath: "smallIntOps.go"},
+			&eapigen.EIGroup{SourcePath: "../cryptoapi/cryptoei.go"},
```

Comment:
```suggestion
			{SourcePath: "bigFloatOps.go"},
			{SourcePath: "bigIntOps.go"},
			{SourcePath: "elrondei.go"},
			{SourcePath: "generateOps.go"},
			{SourcePath: "managedei.go"},
			{SourcePath: "manBufOps.go"},
			{SourcePath: "smallIntOps.go"},
			{SourcePath: "../cryptoapi/cryptoei.go"},
```
The struct literals are implicit here.